src/mpl/src/shm/mpl_shm_mmap.c

```diff
@@ -133,7 +134,7 @@ int MPL_shm_seg_open(MPL_shm_hnd_t hnd, intptr_t seg_sz)
 int MPL_shm_seg_create_and_attach(MPL_shm_hnd_t hnd, intptr_t seg_sz,
                                   char **shm_addr_ptr, int offset)
 {
-    int rc = 0;
+    int rc = -1;
```

Comment:
initialization here really doesn't do anything since we set the value on the next line, so this change is unneeded. in fact, we could probably remove the initialization in a later commit if we wanted.

Reply:
Ok. I will amend the commit and do `return MPL_shm_seg_create_attach_templ(...)` instead.